agent/ecscni/types_windows.go

```diff
@@ -29,8 +29,8 @@ const (
 	// ECSBridgeNetworkName is the name of the HNS network used as ecs-bridge.
 	ECSBridgeNetworkName = "nat"
 	// Constants for creating backoff while retrying setupNS.
-	setupNSBackoffMin      = time.Second * 2
-	setupNSBackoffMax      = time.Second * 6
+	setupNSBackoffMin      = time.Second * 4
```

Comment:
How did you come up with these values?

Reply:
These values are based on practical observations of running multiple tasks with multiple containers in each task.

Reply:
Thanks!

nit: the name `setupNSBackoffMin` being set to `time.Second * 4` is confusing.

Reply:
`setupNSBackoffMin` is the existing constant name being used in the agent.
We have just set that to this value on Windows.